README.md

```diff
@@ -73,8 +73,8 @@ GlobalID::Locator.locate_signed(expiring_sgid.to_s, for: 'sharing')
 # => nil
 ```
 
-**In Rails, an auto-expiry of 1 month is set by default.** You can alter that deal
```

Comment:
Minor: I believe this phrasing is for this reason: https://knowyourmeme.com/memes/i-am-altering-the-deal